apps/events/viewsets.py

```diff
@@ -204,6 +206,21 @@ def set_private_url(self, request):
         return Response(private_url, status=status.HTTP_200_OK)
 
 
+    @action(
+        methods=["get"],
+        detail=True,
+        #permission_classes=[IsAuthenticated],
+        url_path="events_participants",
+        url_name="events_participants_get_list",
+    )
+
+    def list_user_planned_events(self, request, event_id):
+        event = Event.objects.get(id=event_id)
+        queryset= event.participants.all()
+        serializer = CreatorSerializer(queryset, many=True)
+        return Response(serializer.data)
```

Comment:
You forgot to uncomment `permission_classes` and rename the function.
Also, since we are in `EventViewset`, we can use `self.get_object` here to take an event object.
And the last one, why do you use `CreatorSerializer` here?

Reply:
Changes made